src/capture.js

```diff
@@ -9,13 +9,21 @@ function assertActive(id) {
     throw error;
   }
 }
+function assertConnected(elem) {
+  if (!elem.ownerDocument.contains(elem)) {
+    var error = new Error('InvalidStateError');
+    error.name = 'InvalidStateError';
+    throw error;
+  }
+}
 function inActiveButtonState(id) {
   var p = dispatcher.pointermap.get(id);
   return p.buttons !== 0;
 }
 if (n.msPointerEnabled) {
   s = function(pointerId) {
     assertActive(pointerId);
+    assertConnected(this);
```

Comment:
I'm not sure that order matters here, but we should at least be consistent in our two paths. For msPointer, we check active then connected; for everyone else we check connected then active.